### ISSUE TITLE
no lunch 

### ISSUE THREAD

**Issue by ihohenheim** [mmtrt/leagueoflegends#140]
error, pleas is not open game

00c2:err:winediag:MIDIMAP_drvOpen No software synthesizer midi port found, Midi sound output probably won't work
![Captura de pantalla de 2020-07-04 18-57-14](https://user-images.githubusercontent.com/12384243/86522259-214f4080-be29-11ea-850f-e82b224dba56.png)

**Comment by m4rcus25** [comment]
kill all league processes and restart it if the game is not starting.
The Midi sound output error can be ignored completely.

Cheers :)

**Comment by ihohenheim** [comment]
**The Midi sound output error can be ignored completely.**
How, pleas i'm is noob in linux

**Comment by m4rcus25** [comment]
Just ignore it? i dunno, dont do anything about it? I dont know how you ignore stuff, but not giving a damn works really well for me :) The error would usually just appear in the terminal with no consequences or errors during the game.

Have a good day mate :)

**Comment by TheHaricover** [comment]
It look like I got the same issue as ihohenheim. Not the midi port, though.
I have the same logs in console when I run the command 'leagueoflegends".

When the logs get written, I got the LoL logo appearing in the middle of my screen and it disappears at the end of the logs. If I rerun the command, it gets stuck. If I reboot my PC, and run the command again, same issue, the logs are displaying and then nothing.

I don't want to kill the processes as, as described in another issue, it has good chances (if not 100%) to re DL the game.

https://pastebin.pl/view/5412eb98
